proxy/nginx.conf

```diff
@@ -48,9 +56,6 @@ http {
 		location ~ /api/external/nodeNormalization/(.*) {
 			set $url https://nodenormalization-sri.renci.org/;
 			proxy_pass $url$1$is_args$args;
-
-			# Hide long parameters in log
-			access_log /dev/stdout combined_no_query;
```

Comment:
If you remove this line you should remove the definition of combined_no_query at the top of this file.

Reply:
I'm using it on line 27 because we still want to hide long parameters.